yb-voyager/src/srcdb/postgres.go

```diff
@@ -1208,7 +1208,7 @@ func (pg *PostgreSQL) checkPgStatStatementsSetup() (string, error) {
 	}
 
 	if !hasReadAllStats {
-		return "User doesn't have permissions to read pg_stat_statements view, required for detecting Unsupported Query Constructs", nil
+		return "\n" + color.RedString("Missing Permission:") + " User doesn't have the `pg_read_all_stats` grant, required for detecting Unsupported Query Constructs", nil
```

Comment:
Can you post a screenshot of the overall output on CLI? Also when combined with other missing permissions?